src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,34 @@
+@import "../utils/mixins.scss";
+
+@import "../utils/variables.scss";
+
+.calendar {
+  max-width: $day__size * 7 + $flex__gap * 6;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $flex__gap;
+
+  &--position {
+    margin: 0 auto;
+    padding: 20px 0;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    @include square($day__size);
+
+    border: 1px solid #000;
+    background-color: #eee;
+    cursor: pointer;
+    transition: ease-in-out 0.5s;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/222660729-4161374a-2396-4135-9a73-e091806e4ddd.png)

```suggestion
    transition: ease-in-out 0.5s;
```

Reply:
fixed